docs/core/integration-quality-scale/rules/iqs034.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "IQS034 - Dependency transparency"
+---
+
+IQS034 is as follows:
+
+Dependency transparency
+
+## Reasoning
+
+Home Assistant uses a lot of dependencies to work.
+These dependencies will be shipped with new versions of Home Assistant.
+In order for the project to trust the dependencies, we have a set of requirements we want the dependencies to meet.
+
+- The source code of the dependency must be available under an OSI-approved license.
+- The dependency must be available on PyPI.
+- The package published to PyPi should be built and published inside a CI pipeline.
+
+## Additional resources
+
+
```

Comment:
Do we have good sources to link to?

Reply:
Not really, maybe one day we can have some resources on this.